legacy-archive/client/components/main-section/content-view/pdf-exporter/PdfExporter.tsx

```diff
@@ -19,7 +19,9 @@ type Props = {
 export const PdfExporter = ({ content, hidden }: Props) => {
     const { appContext } = useAppState();
     const { basePath } = appContext;
-    const pdfApi = `https://cms-arkiv.ansatt.nav.no${basePath}/pdf`;
+    const pdfApi = document.location.host.includes('localhost')
+        ? `http://localhost:3399${basePath}/pdf`
+        : `https://cms-arkiv.ansatt.nav.no${basePath}/pdf`;
```

Comment:
Vurder å bruk env variabler for å lage url-en, process.env.APP_ORIGIN_INTERNAL f.eks. Trenger kanskje en ny i så fall